frontend/src/employee-mobile-frontend/components/attendances/child-info/ChildSensitiveInfo.tsx

```diff
@@ -0,0 +1,238 @@
+import React from 'react'
+import { Child } from '../../../api/attendances'
+import { useTranslation } from '../../../state/i18n'
+import { ContentAreaWithShadow } from '../../mobile/components'
+import { FixedSpaceColumn } from '../../../../lib-components/layout/flex-helpers'
+import Title from '../../../../lib-components/atoms/Title'
+import CollapsibleSection from '../../../../lib-components/molecules/CollapsibleSection'
+import { P, Label } from 'lib-components/typography'
+import { PlacementType } from '../../../types'
+import styled from 'styled-components'
+
+interface Props {
+  child: Child | null
+}
+
+const KeyValue = styled.div``
+
+const Divider = styled.div`
+  border-bottom: 1px solid #d8d8d8;
+`
+
+const render = (label: string, value: string | null, dataQa: string) => {
+  return value ? (
+    <KeyValue>
+      <Label>{label}</Label>
+      <P fitted data-qa={dataQa}>
+        {value}
+      </P>
+    </KeyValue>
+  ) : null
+}
+
+export default React.memo(function ChildSensitiveInfo({ child }: Props) {
+  const { i18n } = useTranslation()
+
+  const placementTypesToText = (types: PlacementType[]): string => {
+    return types.map((type) => i18n.common.placement[type]).join(',')
+  }
+
+  return (
+    <>
+      {child && (
+        <FixedSpaceColumn spacing={'m'}>
+          <ContentAreaWithShadow
+            opaque={true}
+            paddingHorizontal={'s'}
+            paddingVertical={'m'}
+          >
+            <FixedSpaceColumn alignItems={'center'} spacing={'m'}>
+              <Title>{i18n.attendances.childInfo.header}</Title>
+            </FixedSpaceColumn>
+          </ContentAreaWithShadow>
+
+          <ContentAreaWithShadow opaque={true}>
+            <CollapsibleSection
```

Comment:
I don't know if you used this component intentionally, but it doesn't match the one used in spec.

Reply:
Checking this from UX

Reply:
There is an ongoing discussion how this should be solved. Will be fixed in next PR if needed